tools/bmh_test/run_local_bmh_test_setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -x
+
+set -xe
+
+# Install virtinst
+sudo apt update
+sudo apt install virtinst
+
+# Define and start a virtual network
+virsh -c qemu:///system net-define "tools/bmh_test/bmo_net.xml"
+virsh -c qemu:///system net-start bmo-test-network
+
+# Install libvirt clients and daemon system, python3-pip and pip install vbmc
+sudo apt-get install libvirt-clients libvirt-daemon-system
+sudo apt-get install python3-pip
+pip install vbmc
+
+# Start VBMC
+docker run --name vbmc --network host -d \
+-v /var/run/libvirt/libvirt-sock:/var/run/libvirt/libvirt-sock \
```

Comment:
```suggestion
  -v /var/run/libvirt/libvirt-sock:/var/run/libvirt/libvirt-sock \
```
If command is multi-line, please indent the continuation lines.